scripts/build_demand.py

```diff
@@ -108,25 +119,21 @@ def get_WorldPop_path(
         os.getcwd(),
         "pypsa-earth",
         "data",
-        "Worldpop",
+        "WorldPop",
         f"{three_digits_code.lower()}_ppp_{year}_UNadj_constrained.tif",
     )  # Input filepath tif
 
 
-def create_masked_file(raster_path, shapes_path, output_prefix):
-    """
-    Masks a raster with shapes contained in the GeoJSON file "resources/shapes/microgrid_shapes.geojson" and saves the resulting masked rasters.
-    Parameters:
-    -----------
-    raster_path: str
-        Path to the raster file to mask.
-    shapes_path: str
-        Path to the GeoJSON file containing the shapes to use for masking the raster.
-    output_prefix: str
-        Prefix to use for the output file names. The output files will be named
-        "{output_prefix}_{shape_index}.tif"
-    """
+def estimate_microgrid_population(
+    p, raster_path, shapes_path, sample_profile, output_prefix, output_file
+):
+    # Read the sample profile of electricity demand and extract the column corrisponding to the electric load
+    per_unit_load = pd.read_csv(sample_profile)["0"] / p
+
+    # Dataframe of the load
+    microgrid_load = pd.DataFrame()
 
+    number_microgrids = 3  # len(os.listdir("resources/masked_files"))
```

Comment:
This is a debug value; the number of microgrids and their name shall be provided in the config

Reply:
It was an old thing, I cancelled it